### ISSUE TITLE
Android : Import not working

### ISSUE THREAD

**Issue by tkapias** [Offerel/firefox_syncmarks#5]
Hi,

Do you know if it is supposed to work with the last version of Firefox for android ?

I see the request in nginx :
"GET /seafdav/Syncmarks/bookmarks.json?t=0.5762139675194305 HTTP/2.0" 200 56317 "-" "Mozilla/5.0 (Android 7.0; Mobile; rv:68.0)
 Gecko/68.0 Firefox/68.0"

But there is no change in the bookmarks panel in Firefox.

**Comment by Offerel** [comment]
No, it cant work on Android, since the bookmark API doesn't exist on Firefox Android. Normally it shouldn't be possible to install the AddOn via https://addons.mozilla.org/, because the plugin isn't marked for Android platform. But sometimes the above page is broken or the User-Agent isn't correctly set. In that case, you can install the Addon Android, but it doesn't work.

I fear the bookmark API will never be supported on Firefox Android, since no new features will be added to this platform. Since there are plans to for webextensions on Firefox Preview, i can only hope that bookmark API will be supported there also, but currently, i have no idea, if that will be the case.

The only way is, to use https://github.com/Offerel/SyncMarks as PWA App. Since im not a Android developer, i have no idea how to create a Android app. Maybe, if i have enough time around x-mas, i will try my first steps on Android. If i can get it to work, i can make a app.

So, long story short: Currently it's not easy to se it on Android, only with PWA.
